internal/usage/usage.go

```diff
@@ -15,31 +15,56 @@
-	Profile       = "Profile to use from your configuration file."
-	Members       = "Number of members in the replica set."
-	InstanceSize  = "Tier for each data-bearing server in the cluster."
-	DiskSizeGB    = "Capacity, in gigabytes, of the host’s root volume."
-	Backup        = "If true, uses Atlas Continuous Backups to back up cluster data."
-	MDBVersion    = "MongoDB version of the cluster to deploy."
-	Page          = "Page number."
-	Limit         = "Number of items per page."
-	Username      = "Username for authenticating to MongoDB."
-	Password      = "User’s password."
-	Roles         = "User's roles and the databases or collections on which the roles apply."
-	Comment       = "Optional description of the whitelist entry."
-	Force         = "Don't ask for confirmation."
-	Email         = "User’s email address."
-	FirstName     = "User’s first name."
-	LastName      = "User’s last name."
-	Filename      = "Filename to use"
-	WhitelistIps  = "IP addresses to add to the new user’s whitelist."
-	WhitelistType = `Type of whitelist entry.
-Valid values: cidrBlock|ipAddress`
-	Service = `Type of MongoDB service.
-Valid values: cloud|cloud-manager|ops-manager`
-	Provider = `Name of your cloud service provider.
+	ProjectID                       = "Project ID to use. Overrides configuration file or environment variable settings."
+	OrgID                           = "Organization ID to use. Overrides configuration file or environment variable settings."
+	Profile                         = "Profile to use from your configuration file."
+	Members                         = "Number of members in the replica set."
+	InstanceSize                    = "Tier for each data-bearing server in the cluster."
+	DiskSizeGB                      = "Capacity, in gigabytes, of the host’s root volume."
+	Backup                          = "If true, uses Atlas Continuous Backups to back up cluster data."
+	MDBVersion                      = "MongoDB version of the cluster to deploy."
+	Page                            = "Page number."
+	Limit                           = "Number of items per page."
+	Username                        = "Username for authenticating to MongoDB."
+	Password                        = "User’s password."
+	Roles                           = "User's roles and the databases or collections on which the roles apply."
+	Comment                         = "Optional description of the whitelist entry."
+	Force                           = "Don't ask for confirmation."
+	Email                           = "User’s email address."
+	FirstName                       = "User’s first name."
+	LastName                        = "User’s last name."
+	Filename                        = "Filename to use"
+	WhitelistIps                    = "IP addresses to add to the new user’s whitelist."
+	WhitelistType                   = `Type of whitelist entry. Valid values: cidrBlock|ipAddress`
```

Comment:
@andreaangiolillo just caught this, this used to be multi line helps, please revert the change if move them with the other multiline texts